cobigen-templates/templates-devon4j/src/main/templates/crud_angular_client_app/templates/src/app/${variables.etoName#lower_case}/services/${variables.etoName#lower_case}.service.ts.ftl

```diff
@@ -17,7 +14,7 @@ export class ${variables.etoName?cap_first}Service {
     size: number,
     page: number,
     searchTerms: any,
-    sort: Sort[],
+    sort: any[],
```

Comment:
Why the change from `Sort[]` to `any[]`?

Reply:
Looks like a mistake. Will correct it in the next commit.